libcnb-test/src/build.rs

```diff
@@ -1,62 +1,138 @@
-use cargo_metadata::MetadataCommand;
-use libcnb_package::build::{build_buildpack_binaries, BuildBinariesError};
+use libcnb_common::toml_file::{read_toml_file, TomlFileError};
+use libcnb_data::buildpack::{BuildpackDescriptor, BuildpackId};
+use libcnb_package::buildpack_dependency_graph::{
+    build_libcnb_buildpacks_dependency_graph, BuildBuildpackDependencyGraphError,
+};
 use libcnb_package::cross_compile::{cross_compile_assistance, CrossCompileAssistance};
-use libcnb_package::{assemble_buildpack_directory, CargoProfile};
+use libcnb_package::dependency_graph::{get_dependencies, GetDependenciesError};
+use libcnb_package::output::create_packaged_buildpack_dir_resolver;
+use libcnb_package::package::PackageBuildpackError;
+use libcnb_package::{find_cargo_workspace_root_dir, CargoProfile, FindCargoWorkspaceRootError};
+use std::collections::BTreeMap;
+use std::fs;
 use std::path::PathBuf;
 use tempfile::{tempdir, TempDir};
 
 /// Packages the current crate as a buildpack into a temporary directory.
 pub(crate) fn package_crate_buildpack(
     cargo_profile: CargoProfile,
     target_triple: impl AsRef<str>,
-) -> Result<TempDir, PackageCrateBuildpackError> {
+) -> Result<PackagedBuildpackDir, PackageTestBuildpackError> {
```

Comment:
What up with this rename from `PackageCrateBuildpackError` to `PackageTestBuildpackError`? The previous name matches the error name. With this renaming its no longer the case. Should we rename the function too?

Reply:
I consider `package_crate_buildpack` to be a specialized version of `package_buildpack`.  All the errors are the same between both functions. Also, not all buildpacks are crates (i.e.; meta-buildpacks) which is why I chose to rename to `PackageTestBuildpackError` and don't see the need for the error name to align with the function names here.

Reply:
@Malax given the above, would you be okay with `PackageTestBuildpackError` or `PackageBuildpackError`?

Reply:
`PackageBuildpackError` seems good to me. Thanks for the explanation! :)